README.md

```diff
@@ -1,3 +1,9 @@
+![Build Status](https://img.shields.io/github/workflow/status/awslabs/karpenter/Presubmit)
+![GitHub stars](https://img.shields.io/github/stars/awslabs/karpenter)
```

Comment:
do we need forks and stars since they're at the top right of the screen anyways?

Reply:
Top level stars don't show up on forks, so if I am reading a fork some good to know information may be?